src/client/auth-client.ts

```diff
@@ -26,13 +26,16 @@ import {
   LocalStorage,
   type StoredKey,
 } from './storage.ts';
+import { PostMessageTransport } from '@slide-computer/signer-web';
+import { Signer } from '@slide-computer/signer';
 
 const NANOSECONDS_PER_SECOND = BigInt(1_000_000_000);
 const SECONDS_PER_HOUR = BigInt(3_600);
 const NANOSECONDS_PER_HOUR = NANOSECONDS_PER_SECOND * SECONDS_PER_HOUR;
 
 const IDENTITY_PROVIDER_DEFAULT = 'https://identity.internetcomputer.org';
```

Comment:
Should this be changed to id.ai yet?

Reply:
Not yet, probably. Definitely something to consider, though.